src/main/java/uk/gov/companieshouse/company_appointments/mapper/CompanyAppointmentMapper.java

```diff
@@ -22,152 +22,170 @@
-                .appointedOn(appointedOn == null ? null : appointedOn.toLocalDate())
-                .appointedBefore(StringUtils.isBlank(appointedBefore) ? null : LocalDate.parse(
-                        appointedBefore,
-                        DateTimeFormatter.ofPattern(APPOINTED_BEFORE_DATE_FORMAT)))
-                .resignedOn(resignedOn == null ? null : resignedOn.toLocalDate())
-                .countryOfResidence(isSecretary ? null : companyAppointmentData.getData().getCountryOfResidence())
-                .dateOfBirth(isSecretary ? null : mapDateOfBirth(companyAppointmentData, registerView))
-                .links(mapLinks(companyAppointmentData))
-                .nationality(companyAppointmentData.getData().getNationality())
-                .occupation(companyAppointmentData.getData().getOccupation())
-                .officerRole(OfficerRoleEnum.fromValue(companyAppointmentData.getData().getOfficerRole()))
-                .address(mapAddress(companyAppointmentData))
-                .identification(mapCorporateInfo(companyAppointmentData))
-                .formerNames(mapFormerNames(companyAppointmentData))
-                .name(mapOfficerName(companyAppointmentData))
-                .responsibilities(companyAppointmentData.getData().getResponsibilities())
-                .principalOfficeAddress(mapPrincipalOfficeAddress(companyAppointmentData))
-                .contactDetails(mapContactDetails(companyAppointmentData))
-                .isPre1992Appointment(companyAppointmentData.getData().getIsPre1992Appointment());
-                // TODO: Map person number when switching to delta_appointments collection
-        LOGGER.debug("Mapped data for appointment: " + companyAppointmentData.getId());
+                .appointedOn(appointedOn)
+                .appointedBefore(appointedBefore)
+                .resignedOn(resignedOn)
+                .countryOfResidence(isSecretary ? null : data.getCountryOfResidence())
+                .dateOfBirth(isSecretary ? null
+                        : mapDateOfBirth(companyAppointment.getSensitiveData().getDateOfBirth(), registerView))
+                .links(mapLinks(data.getLinks()))
+                .nationality(data.getNationality())
+                .occupation(data.getOccupation())
+                .officerRole(OfficerRoleEnum.fromValue(data.getOfficerRole()))
+                .address(mapAddress(data.getServiceAddress()))
+                .identification(mapCorporateInfo(data.getIdentification()))
+                .formerNames(mapFormerNames(data.getFormerNames()))
+                .name(mapOfficerName(data))
+                .responsibilities(data.getResponsibilities())
+                .principalOfficeAddress(mapPrincipalOfficeAddress(data.getPrincipalOfficeAddress()))
+                .contactDetails(mapContactDetails(data.getContactDetails()))
+                .isPre1992Appointment(data.getPre1992Appointment())
+                .personNumber(data.getPersonNumber());
```

Comment:
Can we test that person number is mapped each time in ```CompanyAppointmentMapperTest.java```?

Reply:
done, refactored the test too